src/pages/form-validation/verify-on-submit/index.tsx

```diff
@@ -0,0 +1,237 @@
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <Menu />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Verify on Submit
+                    </Typography>
+                </Toolbar>
+            </AppBar>
+
+            <div className={classes.containerWrapper}>
+                <div className={classes.container}>
+                    <Slide
+                        direction="right"
+                        in={showAddDeviceScreen}
+                        mountOnEnter
+                        unmountOnExit
+                        timeout={deviceAdded ? slideAnimationDurationMs : 0}
+                    >
+                        <div>
+                            <Typography variant={'h6'} style={{ marginBottom: theme.spacing(2) }}>
+                                Find Device
+                            </Typography>
+                            <Typography variant={'body1'} style={{ marginBottom: theme.spacing(4) }}>
+                                For the sake of this example, serial number <strong>123</strong> will yield a successful
+                                device search.
+                            </Typography>
+                            <TextField
+                                style={{ width: '100%', height: 72 }}
+                                id={inputId}
+                                label={'Serial Number'}
+                                value={serialNumber}
+                                onChange={onSerialNumberChange}
+                                variant="filled"
+                                error={Boolean(errorMsg)}
+                                helperText={errorMsg}
+                                onKeyUp={(event): void => {
+                                    if (event.key === 'Enter') {
+                                        onSubmit(serialNumber);
+                                    }
+                                }}
+                            />
+
+                            <div className={classes.buttonContainer}>
+                                <Button
+                                    className={classes.submitButton}
+                                    color={'primary'}
+                                    variant={'contained'}
+                                    startIcon={<>{!loading && <Search className={classes.searchIcon} />}</>}
```

Comment:
You shouldn't need this searchIcon class anymore now that you are using the startIcon class (the icon size can be 16, can be 20, not a significant issue here)

Reply:
Ah okay, I've removed the searchIcon class and the default's looking good.